src/main/java/thePackmaster/cards/grandopeningpack/Starring.java

```diff
@@ -0,0 +1,52 @@
+package thePackmaster.cards.grandopeningpack;
+
+import com.evacipated.cardcrawl.mod.stslib.cards.interfaces.StartupCard;
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.animations.VFXAction;
+import com.megacrit.cardcrawl.actions.common.MakeTempCardInDrawPileAction;
+import com.megacrit.cardcrawl.actions.common.MakeTempCardInHandAction;
+import com.megacrit.cardcrawl.actions.utility.ShowCardAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.vfx.combat.ClashEffect;
+import thePackmaster.cards.AbstractPackmasterCard;
+import thePackmaster.util.Wiz;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+import static thePackmaster.util.Wiz.atb;
+
+public class Starring extends AbstractGrandOpeningCard {
+    public final static String ID = makeID("Starring");
+
+    public Starring() {
+        super(ID, 1, CardType.SKILL, CardRarity.COMMON, CardTarget.SELF);
+        baseMagicNumber = magicNumber = 1;
+        baseBlock = block = 8;
```

Comment:
Since i moved the Rarities around a bit, i made Starring a Common card. Because of that, i thought it'd be better to be a Block card, as i already have 2 common attacks in the pack.